<?xml version="1.0"?>
<launch>

    <node pkg="lvr_ros" type="lvr_ros_hdf5_to_msg" name="hdf5_to_msg" output="screen">
        <param name="inputFile" value="/home/jp/pg/data/map-wachsbleiche.h5"/>
    </node>

</launch>
